CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.3.1
+
+* Use public msgpack related API instead of internal its API
```

Comment:
`Use public msgpack APIs instead of its internal APIs` or `Use msgpack's public APIs instead of its internal APIs` reads better because:

 * The pronoun "its" should come before the word "internal"
 * The word "related" sounds a little redundant
 * The word "API" is countable (in most cases, this is not a big deal though)